jitsi-media-transform/src/main/resources/reference.conf

```diff
@@ -30,6 +30,19 @@ jmt {
   dtls {
     // The maximum length of time to wait for a DTLS handshake to complete.
     handshake-timeout = 30 seconds
+
+    // The list of cipher suites to use for DTLS. The names must correspond to the constants in BouncyCastle's
+    // CipherSuite class.
+    cipher-suites = [
+      TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,
+      TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
+      TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,
+      TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
+      TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256,
+      TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256,
+      TLS_DHE_RSA_WITH_AES_128_GCM_SHA256,
+      TLS_DHE_RSA_WITH_AES_256_GCM_SHA384
```

Comment:
This should presumably be updated to the list that @nils-ohlmeier came up with

Reply:
I took this from Nils' latest commit. Do you notice a difference?

Reply:
I missed his "updated cipher list based on Mozilla SSL configurator" commit, so I was remembering the previous one.  This is ok.